src/components/DirectoryRootGrid.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+import { twMerge } from "tailwind-merge";
+import { formatPhoneNumber, formatPhoneNumberIntl } from 'react-phone-number-input';
+
+
+export const directoryRootGridFields = [
+    "dm_directoryChildren.name",
+    "dm_directoryChildren.slug",
+    "dm_directoryChildren.c_addressRegionDisplayName",
+    "dm_directoryChildren.dm_directoryChildrenCount"
+];
+
+
+interface DirectoryRootProps {
+    name?: string;
+    description?: string;
+    directoryChildren?: any;
+    relativePrefixToRoot?: string;
+}
+
+const DirectoryRootGrid = ({ name, description, directoryChildren, relativePrefixToRoot } : DirectoryRootProps) => {
+    let sortedChildren = directoryChildren.sort(function(a:any, b:any) {
```

Comment:
It looks like sortedChildren is not being used. I think it should be removed if that is the case.